server/tserver/src/main/java/org/apache/accumulo/tserver/tablet/Tablet.java

```diff
@@ -164,9 +167,10 @@ public long getDataSourceDeletions() {
   }
 
   private enum CloseState {
-    OPEN, CLOSING, CLOSED, COMPLETE
+    OPEN, REQUESTED, CLOSING, CLOSED, COMPLETE
   }
 
+  private volatile long closeRequestTime = 0;
```

Comment:
This does not need to be volatile because it is only accessed inside of the synchronized block inside initiateClose()

Reply:
There's no penalty for leaving as `volatile`, right? There is only a penalty when you try to access it? Curious if we should leave as-is in case something tries to use it in the future.

Reply:
That is a good question, I wouldn't think it would make much of a difference so it's probably ok. 

 @keith-turner any thoughts on if leaving volatile when currently not needed is a big deal performance penalty wise?

Reply:
Would guess its probably not a performance penalty as synchronized block ensure memory consistency.  If the variable is only accessed in a sync block I usually do not make it volatile because to me volatile implies its ok to access outside of a synch block.  If two or more volatile variables are dependent on each other  then accessing outside of a synch block is very tricky and should probably be avoided.  Volatile is best for independent variables that are safe to access outside of synch.

Reply:
Removed volatile modifier in 3fa4f87